src/SurveyResult/SurveyResult.jsx

```diff
@@ -116,17 +116,14 @@ class SurveyResult extends Component {
   }
 
   componentWillMount() {
+    const qustionId = "5d1a1d16d910160030d04979";
+    const answerId = "5d235426fc81ba2827c5a399";
 
-    const qId = "5d1a1d16d910160030d04979";
-    const aId = "5d1c7bb973589a0030d798ae";
-    const testUrl = "http://mhsbackend.azurewebsites.net/api/v1/questionnaire_sJS/" + qId;
-
-    getAnsweredQuestionnaire(aId)
+    getAnsweredQuestionnaire(answerId)
```

Comment:
@JustinYaaang You are passing the **same** answerID to the function _getAnsweredQuestionnaire_ everytime. So you get the responses for that questionnaire regardless of what you have chosen.

Reply:
The answer is hardcoded atm. We will change that today.